libraries/Microsoft.Bot.Builder.LanguageGeneration/LGOptions.cs

```diff
@@ -0,0 +1,41 @@
+﻿using AdaptiveExpressions;
```

Comment:
Maybe we call this Options to align with Expression lib, or EvaluationOptions, both OK.